client/src/MaterialUI/Component/SimpleSelect/SimpleSelect.js

```diff
@@ -0,0 +1,61 @@
+import React from 'react';
+import {
+  makeStyles,
+  InputLabel,
+  MenuItem,
+  FormControl,
+  Select,
+} from '../../Import/Import';
+
+let minWidth = 300;
+const SimpleSelect = (props) => {
+  console.log(minWidth);
+  const useStyles = makeStyles((theme) => ({
+    formControl: {
+      margin: theme.spacing(1),
+      minWidth: minWidth,
+    },
+    selectEmpty: {
+      marginTop: theme.spacing(2),
+    },
+  }));
+  const itemList = props.itemList;
+  const menuList = itemList.map((i) => {
+    return (
+      <MenuItem key={i.value} value={i.value}>
+        {i.displayValue}
+      </MenuItem>
+    );
+  });
+  const classes = useStyles();
+  const [age, setAge] = React.useState('');
```

Comment:
age should be null by default and not empty string

Reply:
It was part of the library but now fixed/